.cargo/config

```diff
@@ -0,0 +1,8 @@
+[target.x86_64-unknown-linux-gnu]
+# Make NT_GNU_BUILD_ID header generation deterministic.
+#
+# Either `none` or `sha1` values would make the header deterministic, assuming the rest of the build
+# process is deterministic.
+#
+# See https://linux.die.net/man/1/ld (--build-id flag).
+rustflags = ["-C", "link-args=-Xlinker --build-id=none"]
```

Comment:
Do we need something similar in our Bazel config, or are we assuming that Cargo will eventually drive all linking?

Reply:
I was going to ask you exactly this :) which is why I had left #862 assigned to you